packages/devtools-reps/src/reps/stubs/date-time.js

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+let stubs = new Map([]);
```

Comment:
I think we can remove the array inside, my bad for having it in the guide and in the NaN stub